index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Joel's Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+
+</head>
+<body>
+    
+    <div class="wrapper">
+    <header class="main-body">
+        <div class="tools">
+            <nav class="main-nav">
+                <div class="items">
+                    <a href="#">My Logo</a>
+                    <div id="harmburger">
+                        <img alt="harmburger" src="images/harmburger_img.png">
+                    </div>
+                </div>
+                <!-- <div class="col-item">
+                    <div><a href="#portfolio">Portfolio</a></div>
+                    <div><a href="#about">About</a></div>
+                    <div><a href="#contact">Contact</a></div>
+
+                </div> -->
```

Comment:
great work @Joel-100 🤝

- [ ] Kindly remove commented code from your project as this is regarded as a bad practice.